daemon/api_test.go

```diff
@@ -1576,21 +1610,27 @@ func (s *apiSuite) TestDisconnectPlugFailureNoSuchPlug(c *check.C) {
 	c.Check(err, check.IsNil)
 	c.Check(body, check.DeepEquals, map[string]interface{}{
 		"result": map[string]interface{}{
-			"message": `cannot disconnect plug "plug" from snap "producer", no such plug`,
+			"message": `cannot perform the following tasks:
+- Disconnect producer:plug from consumer:slot (cannot disconnect plug "plug" from snap "producer", no such plug)`,
```

Comment:
This is the sort of errors that the ifacestate function should fail upfront with, rather than waiting until the task roundtrips to tell the data is completely bogus. But we can polish that later.